Rename --no-loader-dupecheck option to --no-loader-dupe-check

magnumripper opened the pull request:
Also updates a couple of comments in john.conf to current option syntax

magnumripper committed:
Rename --no-loader-dupecheck option to --no-loader-dupe-check

Also updates a couple of comments in john.conf to current option syntax

solardiz commented on doc/OPTIONS:
```diff
@@ -50,14 +50,14 @@ this option would have the same result as having that word in the gecos
 eg. where you intend to use the --loopback option later instead.
 
---loader-dupecheck=BOOL
+--no-loader-dupecheck
```
Forgot to insert the `-` here.

solardiz commented:
Looks like this is based off `john.conf` prior to my addition of `PerRuleStats`, but GitHub seems happy to merge anyway.

magnumripper commented:
> Looks like this is based off `john.conf` prior to my addition of `PerRuleStats`, but GitHub seems happy to merge anyway.

That should be no problem unless diff contexts start clashing, but I rebased it now that I fixed the other thing.

solardiz commented:
> unless diff contexts start clashing

I actually thought they could - it's nearby lines.
